services/merak-topo/handler/handler.go

```diff
@@ -224,31 +200,54 @@ func Create(k8client *kubernetes.Clientset, topo_id string, aca_num uint32, rack
 		}
 	}
 
-	go Topo_deploy(k8client, aca_image, ovs_image, topo, aca_parameters)
+	if namespace != "default" {
+		nsSpec := &corev1.Namespace{ObjectMeta: metav1.ObjectMeta{Name: namespace}}
+		k8client.CoreV1().Namespaces().Create(context.Background(), nsSpec, metav1.CreateOptions{})
+		utils.Logger.Info("request DEPLOY", "create k8s cluster namespace for new topology deployment", namespace)
+	}
 
-	return nil
+	go Topo_deploy(k8client, aca_image, ovs_image, topo, aca_parameters, topoPrefix, namespace)
+
+	elaps2 := time.Since(start1)
+
+	if err_flag == 1 {
+		err_return = errs
+		utils.Logger.Info("request DEPLOY", "Fails: check merak-topo logs", elaps2)
+	} else {
+		err_return = nil
+		utils.Logger.Info("request DEPLOY", "Complete: Return compute nodes and k8s cluster information to Scenario Manager (in second)", elaps2)
+	}
```

Comment:
@zzengcs Is that really need to use `err_flag` in the end of function to identity all errors from above?

Reply:
@cj-chung using err_flag is to give user the readable error info by skipping the details. The detailed error msgs are recorded by log. If don't use it, any suggestions?